internal/pkg/service/appsproxy/proxy/server.go

```diff
@@ -27,6 +30,7 @@ func StartServer(ctx context.Context, d dependencies.ServiceScope) error {
 
 	// Start HTTP server
 	srv := &http.Server{Addr: cfg.API.Listen, Handler: handler, ReadHeaderTimeout: readHeaderTimeout}
+	srv.ErrorLog = log.NewStdErrorLogger(d.Logger().WithComponent("http-server"))
 	proc := d.Process()
```

Comment:
Added missing logger. 

We already check there is no error after each test, but errors directly from the http server were missing.